metashare/api/serializers.py

```diff
@@ -237,6 +280,20 @@ def get_pr_url(self, obj) -> Optional[str]:
             return f"https://github.com/{repo_owner}/{repo_name}/pull/{pr_number}"
         return None
 
+    def update(self, instance, validated_data):
+        originating_user_id = getattr(
+            getattr(self.context.get("request"), "user", None), "id", None
+        )
```

Comment:
@wlonk Since this is a side-effect, I actually think `originating_user_id=None` would make more sense.

Reply:
OK!